src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,117 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
+@SuppressWarnings("unchecked")
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double NEW_ARRAY_CAPACITY_FACTOR = 1.5;
+    private int size;
+    private Object[] elementData;
```

Comment:
You need to use the parametrize type
```suggestion
    private T [] elementData;
```